### DISCUSSION TITLE
Use files in a repeater with a title / How to translate a file title

### DISCUSSION THREAD

**Discussion by flayks** [directus/directus#2835] [Archive]
Hey there!
I'm developing a website with two languages and I'm facing this issue that I need to translate a title for a set of images, then repeatable. One idea was to create a Repeater field and add a File field and a Text field but as we can't use a File in a repeater, that's not gonna work.
So I was wondering how I can achieve that? It doesn't seem that we can translate file titles, which is a bummer.
Any idea is welcome because I feel like it's not possible the same way ACF on Wordpress would work (add any field together as a group that you can repeat X times)
Thanks ✌️

**Comment by flayks** [comment]
Any hope for that usage in the wait of the version 9?

**Comment by benhaynes** [comment]
Merging this into: https://github.com/directus/directus/discussions/3001

Though this should be possible with a relational collection that holds an image and a translation field for the titles.
